gradle.properties

```diff
@@ -2,4 +2,4 @@ nexusUrl=http://igo-ci01.mskcc.org:8081
 nexusUrlReleases=http://igo-ci01.mskcc.org:8081/repository/maven-releases
 nexusUsername=admin
 nexusPassword=
-target=dev
+target=
```

Comment:
Now that all the settings are the same we could change the .jar so the release is the same for igo-lims04 & igolims